seleniumByTechnocredits/src/seleniumPractise/vrushali/RegistrationForm.java

```diff
@@ -0,0 +1,206 @@
+
+		// validation for other email id
+
+		RegistrationForm.validation(driver);
+		Alert alertOtherEmail = driver.switchTo().alert();
+		String a3 = alertOtherEmail.getText();
+		alertOtherEmail.accept();
+
+		if (a3.equals("Please enter email id")) {
+			System.out.println("Please enter email id");
+
+		}
+		driver.findElement(By.xpath("//input[@id='useremail'][@name='useremail']")).sendKeys("vrusali@g.com");
+
+		// validations for city
+		RegistrationForm.validation(driver);
+		Alert alertCity = driver.switchTo().alert();
+		String a4 = alertCity.getText();
+		alertCity.accept();
+
+		if (a4.equals("Please enter City")) {
+			System.out.println("Please enter City");
+		}
+		driver.findElement(By.xpath("//input[@id='usercity']")).sendKeys("pune");
+
+		// Validations for company name
+		RegistrationForm.validation(driver);
+		Alert alertCompany = driver.switchTo().alert();
+		String a5 = alertCompany.getText();
+		alertCompany.accept();
+
+		if (a5.equals("Please enter your current organization")) {
+			System.out.println("Organization can't be blank, Please enter your current organization");
+
+		}
+		driver.findElement(By.xpath("//input[@id='organization'] | //input[@name = 'organization']")).sendKeys("TCS");
+
+		// Validations for radio button
+		WebElement genderRadioButton = driver.findElement(By.xpath("//input[@id='radio-02']"));
+		if (genderRadioButton.isDisplayed()) {
+			if (!genderRadioButton.isSelected()) {
+				genderRadioButton.click();
+			}
+		}
+
+		// validation for User name
+
+		RegistrationForm.validation(driver);
+		Alert alertUserName = driver.switchTo().alert();
+		String a6 = alertUserName.getText();
+		alertUserName.accept();
+
+		if (a6.equals("Username is mandatory field.")) {
+			System.out.println("Username is mandatory field. Enter user name");
+		}
+		WebElement e1 = driver.findElement(By.id("usernameReg"));
+		e1.clear();
+		e1.sendKeys("Vrush");
+		RegistrationForm.validation(driver);
+		Alert alertUserName1 = driver.switchTo().alert();
+		String a7 = alertUserName1.getText();
+		alertUserName1.accept();
+
+		if (a7.equals("Username length should be greater then 5 characters.")) {
+			System.out.println("Username length should be greater then 5 characters.");
+		}
+		e1.clear();
+		e1.sendKeys("Vrushali");
+		RegistrationForm.validation(driver);
+		Alert alertUserName2 = driver.switchTo().alert();
+		String a8 = alertUserName2.getText();
+		alertUserName2.accept();
+
+		// validation for password field
+		RegistrationForm.validation(driver);
+		Alert alertPwd = driver.switchTo().alert();
+		String a9 = alertUserName.getText();
+		alertPwd.accept();
+
+		if (a9.equals("password is mandatory field.")) {
+			System.out.println("password is mandatory field.. Enter password");
+		}
+
+		WebElement pd = driver.findElement(By.id("passwordReg"));
+		pd.clear();
+		pd.sendKeys("test");
+
+		RegistrationForm.validation(driver);
```

Comment:
RegistrationForm.validation(driver);
		Alert alertPwd1 = driver.switchTo().alert();
		String a10 = alertUserName1.getText();
		alertPwd1.accept();

The above set of code is being repeated rather than writing every time you have to create a method and call it every time.

Reply:
Sure will keep in mind and will follow this approach from next assignments. @krishnakTechnocretids